packages/lab/src/date-picker/DatePicker.tsx

```diff
@@ -81,6 +87,21 @@ export interface DatePickerProps
    * If `true`, the component is read only.
    */
   readOnly?: boolean;
+  /**
+   * Validation status.
+   */
+  validationStatus?: "error" | "warning" | "success";
+  /**
+   * Callback fired when the selected daate change.
+   */
+  onSelectionChange?: (
+    event: SyntheticEvent,
+    selectedDate?: DateValue | { startDate?: DateValue; endDate?: DateValue }
+  ) => void;
+  /**
+   * Callback fired when the input value change.
+   */
+  onInputValueChange?: (event: SyntheticEvent, newInput: string) => void;
```

Comment:
May need to look at Dropdown/combo, on `onSelectionChange` vs `onChange` vs `onInputValueChange`